docs/Doxyfile.config

```
@@ -863,7 +863,8 @@ INPUT                  = "@MARKDOWN_INPUT@" \
                          "@OpenVINO_SOURCE_DIR@/src/core/shape_inference/include/" \
                          "@OpenVINO_SOURCE_DIR@/src/frontends/common/include/" \
                          "@OpenVINO_SOURCE_DIR@/src/inference/dev_api/" \
-                         "@OpenVINO_SOURCE_DIR@/src/inference/include/"
+                         "@OpenVINO_SOURCE_DIR@/src/inference/include/" \
+                         "@OpenVINO_SOURCE_DIR@/src/src/bindings/c/include/openvino/c/"
```

Looks like path is incorrect, I propose:
```suggestion
                         "@OpenVINO_SOURCE_DIR@/src/bindings/c/include/"
```